opentelemetry-sdk/tests/trace/test_trace.py

```diff
@@ -806,6 +806,20 @@ def error_status_test(context):
             .start_as_current_span("root")
         )
 
+    def test_record_error(self):
+        span = trace.Span("name", mock.Mock(spec=trace_api.SpanContext))
+        try:
+            raise ValueError("invalid")
+        except ValueError as err:
+            span.record_error(err)
```

Comment:
```suggestion
        span.record_error(ValueError("invalid")
```

Reply:
this won't set a valid `error.stack` attribute on the event